app/controllers/blogs_controller.rb

```diff
@@ -0,0 +1,59 @@
+class BlogsController < ApplicationController
+
+  before_action :logged_in_user, only: [:new, :edit, :index, :show, :create,]
+
+
+  def index
+
+  end
+
+
+  def show
+
+  end
+
+
+  def edit
+
+  end
+
+  def new
+  	@blog = Blog.new
+  end
+
+  def create	
+  	@blog = Blog.new(params.require(:blog).permit(:title,:caption,:description))
+  	@blog.user_id = current_user.id
```

Comment:
Use the active record on creating relationship values. Example, since blogs belongs to user you can just declare this as current_user.blogs.new(params[:blogs]) @vincelaunchlabs